scripts/readmeFiles.js

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env node
+const fs = require("fs");
+// eslint-disable-next-line import/no-extraneous-dependencies
+const shell = require("shelljs");
+// eslint-disable-next-line import/no-extraneous-dependencies
+const reactDocs = require("react-docgen");
+
+const skipPackages = ["helpers", "Stylers", "Tokens", "Guard", "Icon", "Overlay", "Select"];
```

Comment:
this package might lack a `readme` files or require specific treatment. Will investigate later, for now, are being skipped.